.circleci/config.yml

```diff
@@ -853,18 +855,12 @@ workflows:
             - hold-build-logspout
       - hold-bake-gcp-dev-image:
           type: approval
-          filters:
```

Comment:
this is so we can run image bakes from anywhere. should be safe to run from any branch plus there's a hold as well.